numba/tests/test_extending.py

```diff
@@ -1576,13 +1692,70 @@ def passthru(x):
         with self.assertRaises(ValueError) as raises:
             passthru(self.dyn_type)
         self.assertIn(
-            "cannot do x > 0",
-            str(raises.exception),
+            "cannot do x > 0", str(raises.exception),
         )
 
 
-class TestMisc(TestCase):
+def with_objmode_cache_ov_example(x):
+    # This is the function stub for overloading inside
+    # TestCachingOverloadObjmode.test_caching_overload_objmode
+    pass
+
+
+class TestCachingOverloadObjmode(TestCase):
```

Comment:
Suggest mixing with `SerialMixin` here to see if that fixes the problem on windows.